.github/workflows/markdown-links-verifier.yml

```diff
@@ -0,0 +1,14 @@
+name: Markdown links verifier
+on: push
+
+jobs:
+  validate_links:
+    name: Markdown links verifier
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v1
+
+    - name: Validate links
+      uses: Youssef1313/markdown-links-verifier@v0.0.5
```

Comment:
I'm not sure of your policy of using such GH Actions. Let me know if this shouldn't be used or if it requires any action to be used.